.travis.yml

```diff
@@ -237,6 +237,41 @@ jobs:
         - COMPILER="ccache g++"
       script: echo "This is coverity build. No need for tests."
 
+    - stage: Extended checks
+      env:
+        - NAME="sanitized build"
+      if: type = cron
+      os: linux
+      sudo: required
```

Comment:
Is sudo really required? I don't see what you are doing that does need that, and I think adding the requirement for sudo means we get slower job startup? (i.e. sudo requires a full VM that travis estimate as a 30-50 second startup, where as sudo not required just means firing up a container which is 1-6s). Only other consideration is that sudo-required machines get more RAM allocation, according to: https://docs.travis-ci.com/user/reference/overview/

Reply:
I'll try without sudo and see how it goes